media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-module
+
+DESCRIPTION="Set of utilities to manage TV listings stored in the XMLTV format"
+HOMEPAGE="http://xmltv.org"
+SRC_URI="https://github.com/XMLTV/xmltv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+#KEYWORDS="~amd64 ~arm ~ppc ~x86 ~x86-linux"
+### Temporary until I figure out how to get it fixed!
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+IUSE="ar ch-search dk dtvla eu-dotmedia eu-epgdata eu-xmltvse fi fi-sv fr
+huro il is it na-dd na-dtv na-tvmedia pt-meo pt-vodafone se-swedb se-tvzon tr
+tv-check tv-combiner tv-pick-cgi uk-bleb uk-tvguide zz-sdjson zz-sdjson-sqlite"
+
+RDEPEND="
+	>=dev-lang/perl-5.8.3
+	>=dev-perl/Date-Manip-5.42
+	dev-perl/File-Slurp
+	dev-perl/JSON
+	>=dev-perl/libwww-perl-5.65
+	virtual/perl-Memoize
+	>=virtual/perl-Storable-2.04
+	dev-perl/TermReadKey
+	>=dev-perl/XML-Parser-2.34
+	dev-perl/XML-TreePP
+	>=dev-perl/XML-Twig-3.28
+	>=dev-perl/XML-Writer-0.6
+
+	virtual/perl-IO-Compress
+	>=dev-perl/Lingua-Preferred-0.2.4
```

Comment:
You could also go through this complete list and update version requirements. I checked few and they are always satisfied in ::gentoo tree.

Also please sort the list.

Reply:
Checked all version requirement and all are met in tree so I removed them in ebuild.